app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_list = []
+    for person in people:
+        new_person = Person(person["name"], person["age"])
+        new_list.append(new_person)
```

Comment:
```suggestion
        new_list.append(Person(person["name"], person["age"]))
```

It's redundant variable